openreview/conference/templates/recommendationWebfield.js

```diff
@@ -23,112 +20,19 @@ function main() {
-
-    var notesP;
-    if (noteNumbers.length) {
-      var noteNumbersStr = noteNumbers.join(',');
-
-      notesP = Webfield.getAll('/notes', {
-        invitation: BLIND_SUBMISSION_ID, number: noteNumbersStr, details: 'tags'
-      }).then(function(allNotes) {
-        return allNotes.map(function(note) {
-          note.details.tags = note.details.tags.filter(function(tag) {
-            return tag.tauthor;
-          });
-          return note;
-        });
-      });
-    } else {
-      notesP = $.Deferred().resolve([]);
-    }
-
-    var tagInvitationsP = Webfield.getAll('/invitations', {
-      regex: CONFERENCE_ID + '/Paper.*/-/Recommendation', tags: true, invitee: true
-    });
-
-    return $.when(notesP, tagInvitationsP);
-  });
-}
-
-
-// Util functions
-function getPaperNumbersFromGroups(groups) {
-  return _.filter(_.map(groups, function(group) {
-    return getNumberFromGroup(group.id, 'Paper');
-  }), _.isInteger);
+  return $.Deferred().resolve();
 }
 
-function getNumberfromGroup(groupId, name) {
-  var tokens = groupId.split('/');
-  var paper = _.find(tokens, function(token) {
-    return _.startsWith(token, name);
-  });
-
-  if (paper) {
-    return parseInt(paper.replace(name, ''), 10);
-  } else {
-    return null;
-  }
-};
-
 
 // Display the recommend interface populated with loaded data
-function renderContent(notes, tagInvitations) {
-
-  // Nothing to dispay
-  if (!notes.length) {
-    $('#notes').empty();
-    $('#notes').append('<p class="empty-message">You have no assigned papers at this time.</p>');
-    return;
-  }
-
-  // Set up tabs
-  var sections = [{
-    heading: 'Your Assigned Papers',
-    id: 'your-assigned-submissions',
-  }];
-
-  Webfield.ui.tabPanel(sections, {
-    container: '#notes',
-    hidden: true
-  });
-
-  // Your Assigned Papers tab
-  var submissionListOptions = {
-    pdfLink: true,
-    showContents: true,
-    showTags: true,
-    tagInvitations: tagInvitations,
-    container: '#your-assigned-submissions'
-  };
+function renderContent() {
 
-  Webfield.ui.submissionList(notes, {
-    heading: null,
-    container: submissionListOptions.container,
-    search: {
-      enabled: true,
-      localSearch: true,
-      subjectAreas: SUBJECT_AREAS,
-      subjectAreaDropdown: 'basic',
-      onResults: function(searchResults) {
-        Webfield.ui.searchResults(searchResults, submissionListOptions);
-        Webfield.disableAutoLoading();
-      },
-      onReset: function() {
-        Webfield.ui.searchResults(notes, submissionListOptions);
-      }
-    },
-    displayOptions: submissionListOptions,
-    fadeIn: false
-  });
+  var params = EDGE_BROWSER_PARAMS.replace('{userId}', user.profile.id);
+  var browseUrl = window.location.origin + '/edge/browse?' + decodeURIComponent(params);
 
-  $('#notes > .spinner-container').remove();
-  $('#notes .tabs-container').show();
+  $('#notes').empty().append('<a href="' + browseUrl + '" target="_blank">Go to reviewers recommendation</a>');
```

Comment:
Rather than a link I would like this to be a big button that just says "Recommend Reviewers"

Reply:
Where are the instructions?

Reply:
webfield.py(214)